pkg/detectors/postman/postman.go

```diff
@@ -47,31 +52,50 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 
 		if verify {
-			req, err := http.NewRequestWithContext(ctx, "GET", "https://api.getpostman.com/collections", nil)
-			if err != nil {
-				continue
-			}
-			req.Header.Add("x-api-key", resMatch)
-			res, err := client.Do(req)
-			if err == nil {
-				defer res.Body.Close()
-				if res.StatusCode >= 200 && res.StatusCode < 300 {
-					s1.Verified = true
-				} else {
-					// This function will check false positives for common test words, but also it will make sure the key appears 'random' enough to be a real key.
-					if detectors.IsKnownFalsePositive(resMatch, detectors.DefaultFalsePositives, true) {
-						continue
-					}
-				}
-			}
+			client := s.getClient()
+			isVerified, verificationErr := verifyPostman(ctx, client, resMatch)
```

Comment:
Question for the team: is it worthwhile to update the alchemy template so that it implements `verify{x}` functions?

Reply:
good call. i vote yes. @0x1 what do you think?

Reply:
it might even be worth doing it in a way that facilitates the future creation of some kind of `Verify()` interface method

Reply:
i'd vote yes - it feels a lot cleaner and maintainable to me than current setup

Reply:
adding [ticket](https://trufflesecurity.atlassian.net/browse/THOG-1470) to track this